common/interfaces.ts

```diff
@@ -102,27 +103,30 @@ export type MapFormValue = MapEntry[];
 
 export type MapArrayFormValue = MapFormValue[];
 
-export type TemplateVar = {
+export type ExpressionVar = {
   name: string;
   transform: string;
 };
 
 export type Transform = {
   transformType: TRANSFORM_TYPE;
-  value: string;
+  value?: string;
   // Templates may persist their own set of nested transforms
   // to be dynamically injected into the template
-  nestedVars?: TemplateVar[];
+  nestedVars?: ExpressionVar[];
 };
 
 export type InputMapEntry = {
   key: string;
   value: Transform;
 };
+export type OutputMapEntry = InputMapEntry;
```

Comment:
Yes, for future extensibility, they will likely split off over time. Easier to keep standalone to prevent refactoring a generically-named constant later on.